video/out/cocoa_common.m

```diff
@@ -574,7 +574,6 @@ int vo_cocoa_config_window(struct vo *vo)
         if (!s->embedded && s->window) {
             if (reset_size)
                 queue_new_video_size(vo, width, height);
-            vo_cocoa_fullscreen(vo);
```

Comment:
I am not sure, maybe @wm4 can tell more, but other backends call it unconditionally at start